app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -374,7 +376,7 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService)(implic
 
   }
 
-  override def overallReportNotWithdrawn(frameworkId: String): Future[List[Report]] =
+  override def overallReportNotWithdrawn(frameworkId: String): Future[List[CandidateProgressReport]] =
```

Comment:
If the report is named "Candidate Progress Report" I think is a good idea to rename the method to something like candidateProgressReportNotWithdrawn.